### PR TITLE
Fix installation playbook indentation

### PR DESCRIPTION
**Pull request by mayaCostantini**
Fix indentation for the `pulp_ansible.yml` installation playbook in the docs.

[noissue]

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -17,19 +17,19 @@ Create your pulp_ansible.yml playbook to use with the installer:
...
+      pulp_settings:
+        secret_key: << YOUR SECRET HERE >>
+        content_origin: "http://{{ ansible_fqdn }}"
+      pulp_default_admin_password: << YOUR PASSWORD HERE >>
+      pulp_install_plugins:
+        pulp-ansible: {}
+    roles:
+      - role: pulp.pulp_installer.pulp_all_services
+        environment:
+          DJANGO_SETTINGS_MODULE: pulpcore.app.settings
```
Comment (mdellweg) [inline docs/installation.rst:32]:
```suggestion
    environment:
      DJANGO_SETTINGS_MODULE: pulpcore.app.settings
```

I cannot find role level environment in the ansible docs...

Comment (mdellweg) [reply docs/installation.rst:32]: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_environment.html

Comment (mayaCostantini) [reply docs/installation.rst]: Right, thank you. Fixed in 66c9bd0b33976b3a4a4707962eae357bed027bf8

**Comment by gerrod3** [comment]
@mayaCostantini Can you update your commit message to include `[noissue]` at the end and squash it into one commit?

**Comment by mayaCostantini** [comment]
Thanks @gerrod3, adjusted :+1:

**Review by mdellweg — APPROVED** [review]

**Pull request merged**